composer.json

```diff
@@ -15,14 +15,15 @@
         }
     ],
     "require": {
-        "composer/installers": "^1.9",
+        "composer/installers": "^2.3",
         "cweagans/composer-patches": "^1.7",
-        "drupal/core-composer-scaffold": "^11",
-        "drupal/core-project-message": "^11",
-        "drupal/core-recommended": "^11",
-        "drupal/lupus_decoupled": "1.x-dev",
-        "drupal/rest_log": "^2.3",
+        "drupal/core-composer-scaffold": "^11.1",
+        "drupal/core-project-message": "^11.1",
+        "drupal/core-recommended": "^11.1",
+        "drupal/drupal_cms_starter": "^1.0@RC",
```

Comment:
I'm not conviced this shoudl be here. the repo is a decoupled-project setup, not a CMS demo. So far we added a few modules we support as well, but adding the whole of drupal-cms will be too much. Not sure how to handle best, let's clarify the goal and then see how to best get there!